packages/router/test/regression_integration.spec.ts

```diff
@@ -267,10 +272,74 @@ describe('Integration', () => {
          advance(fixture);
 
          expect(location.path()).toEqual('/');
-         const urlChanges = ['replace: /', 'hash: /one', 'replace: /'];
-         expect(location.urlChanges).toEqual(urlChanges);
+         expect(fixture.nativeElement.innerHTML).toContain('one');
        }));
   });
+
+  describe('#43447, #43446', () => {
```

Comment:
I'd propose extending the `describe` block description, so that we can get the high-level idea without looking at the mentioned issues/PRs.

```suggestion
  describe('duplicate navigation handling (#43447, #43446)', () => {
```